Tryouts/Core/Abstractions/Modules/StartupType.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace MorganStanley.ComposeUI.Tryouts.Core.Abstractions.Modules;
+
+public enum StartupType
```

Comment:
Can you please explain what do these values represent?

Reply:
Added comments